ui/shared/components/panel/family/Family.jsx

```diff
@@ -111,13 +121,13 @@ const FAMILY_FIELD_RENDER_LOOKUP = {
   },
   [FAMILY_FIELD_CODED_PHENOTYPE]: { component: SingleFieldView, canEdit: true },
   [FAMILY_FIELD_MONDO_ID]: {
-    component: SingleFieldView,
+    ...MONDO_FIELD,
     canEdit: true,
-    fieldDisplay: value => (
-      <a target="_blank" rel="noreferrer" href={`http://purl.obolibrary.org/obo/MONDO_${value.replace('MONDO:', '')}`}>
-        {value}
-      </a>
-    ),
+  },
+  [FAMILY_FIELD_DISCOVERY_MONDO_ID]: {
+    ...MONDO_FIELD,
+    internal: true,
+    canEditFamily: ({ discoveryTags }) => discoveryTags?.length > 0,
```

Comment:
What does it mean in terms of the status of the case if it has discovery tags? is it solved if there are discoveryTags?

Reply:
yes, discovery tags mean that someone added at tag to a variant indicating that it is (at least part of) the solve. Also, the post discovery OMIM and MONDO numbers are used for reporting purposes, and they are included per discovery variant, so there is no purpose of having them at all if theres no discovery variant